components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -306,6 +306,11 @@ IB_DESIGNABLE
  */
 @property(nonatomic) CGFloat maximumHeight;
 
+/**
+ Default is YES.
```

Comment:
When this is enabled, the flexible header will assume that minimumHeight and maximumHeight both include the safe area. For example, a header whose maximum content height should be `200` might set `220` (`200 + 20`) as the maximumHeight.

When this is disabled, the flexible header will assume that the provided minimumHeight and maximumHeight do not include the safe area insets. For example, a header whose maximum content height should be `200` would set `200` as the maximumHeight.